### PR TITLE
refactor: FocusId types were clashing

### PR DESCRIPTION
**Pull request by driskull**
refactor: FocusId types were clashing

### CODE REVIEW

File: src/components/calcite-popover/resources.ts [hunk 1]
```diff
@@ -13,3 +13,5 @@ export const TEXT = {
 export const POPOVER_REFERENCE = "data-calcite-popover-reference";
 export const ARIA_CONTROLS = "aria-controls";
 export const ARIA_EXPANDED = "aria-expanded";
+
+export type PopoverFocusId = "close-button";
```
Comment (jcfranco) [inline src/components/calcite-popover/resources.ts:17]: Can we drop the component from all focus ID type names? It's already implied by the module.

**Review by jcfranco — APPROVED** [review]
Just had a comment about simplifying the type names. Other than that, LGTM!

File: src/components/calcite-popover/resources.ts (hunk 1 above)
Comment (driskull) [reply src/components/calcite-popover/resources.ts:17]: @jcfranco it was like that but the typings were renaming one of them to Focus1.

Comment (jcfranco) [reply src/components/calcite-popover/resources.ts:17]: > the typings were renaming one of them to FocusId

I didn't follow that. Can you elaborate?

Comment (driskull) [reply src/components/calcite-popover/resources.ts:17]: Yeah, when I have two exported FocusId interfaces, it does this:

```

import { FocusId } from "./components/calcite-pick-list/shared-list-logic";
--
31 | import { ICON_TYPES } from "./components/calcite-pick-list/resources";
32 | import { FocusId as FocusId1 } from "./components/calcite-popover/calcite-popover";


```

https://unpkg.com/browse/@esri/calcite-components@1.0.0-next.122/dist/types/components.d.ts

Then it causes issues with using the Popover cus the typings are wrong.

Comment (jcfranco) [reply src/components/calcite-popover/resources.ts:17]: Updating based on our side conversation, this seems like a Stencil compiler issue. Looks like named imports do not get assigned properly for all cases. Renaming the imports is a good workaround for now. Once fixed, we can revisit simplifying the type name.

**Pull request merged**